pkg/cloudprovider/fake/instancetype.go

```diff
@@ -194,3 +197,10 @@ func PriceFromResources(resources v1.ResourceList) float64 {
 	}
 	return price
 }
+
+func constraints(capacityType, zone string) map[string]string {
+	return map[string]string{
+		v1beta1.CapacityTypeLabelKey: capacityType,
+		v1.LabelTopologyZone:         zone,
+	}
+}
```

Comment:
you define this same helper function in multiple places maybe we can put it in our helpers?